Dockerfile

```diff
@@ -1,22 +1,25 @@
 # This file is part of REANA.
-# Copyright (C) 2017, 2018, 2020 CERN.
+# Copyright (C) 2017, 2018, 2020, 2021 CERN.
 #
 # REANA is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-FROM ubuntu:16.04
+FROM rabbitmq:3.8.9-management
```

Comment:
Any special need to pin 3.8.9?  I'm seeing 3.8.17 already...

If we don't need a given minor version, we could even use 3.8 generally.  (Provided all plugins are compatibly with major.minor.patchlevel numbering schema.)

Reply:
Changed it to more generic `rabbitmq:3.8-management` image. It should be fine with all the plugins, since all of them ship together with `RabbitMQ`, except `rabbitmq-delayed-message-exchange` which [targets RabbitMQ 3.8.x](https://github.com/rabbitmq/rabbitmq-delayed-message-exchange)